compose/foundation/foundation/src/jsNativeMain/kotlin/androidx/compose/foundation/text/StringHelpers.jsNative.kt

```diff
@@ -16,7 +16,18 @@
 
 package androidx.compose.foundation.text
 
-internal actual fun StringBuilder.appendCodePointX(codePoint: Int): StringBuilder =
-    TODO("Implement native StringBuilder.appendCodePointX")
-    // TODO: there is an appendCodePoint in K/N stdlib, but it is internal.
```

Comment:
Do I remember correctly that there was a ticket to consider making it public in the stdlib?

Reply:
I don't know :)
Maybe @igordmn may know?

Reply:
anyway, it's not really important for this PR :)
I just thought that if there was a ticket, then maybe we can link it here in the comment.

Reply:
> Maybe @igordmn may know?

You are who reported the issue :)

https://youtrack.jetbrains.com/issue/KT-52336/Different-behavior-on-JVM-and-Native-in-stringBuilderappendcharArray-0-1

Reply:
> > Maybe @igordmn may know?
> 
> You are who reported the issue :)
> 
> https://youtrack.jetbrains.com/issue/KT-52336/Different-behavior-on-JVM-and-Native-in-stringBuilderappendcharArray-0-1

It's another one issue.
In this comment we discuss internal StringBuilder.**appendCodePoint** (because we want use it)

Reply:
Ah, got it.